src/main/java/com/CezaryZal/api/shortday/ShortDayService.java

```diff
@@ -0,0 +1,46 @@
+package com.CezaryZal.api.shortday;
+
+import com.CezaryZal.exceptions.ShortDayNotFoundException;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@Service
+public class ShortDayService {
+
+    private ShortDayRepository shortDayR;
+
+    public ShortDayService(ShortDayRepository shortDayR) {
+        this.shortDayR = shortDayR;
+    }
+
+    public ShortDay getShortDayById(Long id) {
+        return shortDayR.findById(id)
+                .orElseThrow(() -> new ShortDayNotFoundException("Short day not found by id"));
+    }
+
+    public List<ShortDay> getShortDaysByDateAndUserId(String inputDate, Long userId) {
+        LocalDate localDateMin = LocalDate.parse(inputDate).minusDays(30);
+        LocalDate localDateMax = LocalDate.parse(inputDate).plusDays(30);
+        List<ShortDay> listShortDay = shortDayR.findAllByUserIdAndDateBetween(userId, localDateMin, localDateMax);
+
+        return listShortDay;
+    }
+
+    public List<ShortDay> getAll() {
+        return (List<ShortDay>) shortDayR.findAll();
+    }
+
+    public void updateShortDay(ShortDay shortDay) {
+        shortDayR.save(shortDay);
```

Comment:
raczej dodaje się to słówko this przed zmienną z pola klasy jeżeli z niej korzystasz, jest dużo czytelniej
this.shortDay.....